### PR TITLE
Test

### PR DESCRIPTION
**Pull request by taylorjosephgriffin**

### CODE REVIEW

File: register.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  cashRegister.scan("eggs",1);
+  cashRegister.scan("milk",1);
+  cashRegister.scan("magazine",3);
+
+  cashRegister.applyStaffDiscount(sally);
+
+  console.log(cashRegister.printReciept());
+;});
+
+  module.exports = function (inventory) {
```
Comment (walston) [inline register.test.js]: This function and export belongs in register.js not the test file

File: register.js [hunk 2]
```diff
@@ -1,16 +1,12 @@
-function StaffMember(name,discountPercent){
+  function StaffMember(name,discountPercent){
```
Comment (walston) [inline register.js]: move this into test.js

File: register.js [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
   this.name = name;
   this.discountPercent = discountPercent;
 }
 
-var sally = new StaffMember("Sally",5);
```
Comment (walston) [inline register.js]: move these staff members into test.js too

**Comment by taylorjosephgriffin** [comment]
merge

**Pull request merged**